plugins/inputs/vsphere/README.md

```diff
@@ -771,7 +771,6 @@ For example:
   cluster_metric_exclude = ["*"]
   
   vsan_metric_include = [ "summary.*" ]
-  vsan_metric_exclude = [ ]
```

Comment:
Why did you remove it again?

Reply:
Added back the vsan_metric_exclude changes. Missed while checking DisconnectedServersBehavior build error.